### PR TITLE
Figaro

### PR DESCRIPTION
**Pull request by Arique1104**
**Details:** This adds the figaro and faraday gems along with the application.yml file for the api key.

**Improvements:** (issues/bugs/refactoring?)

- SimpleCov Model (%): 7 / 28 LOC (25.0%)
- SimpleCov Feature (%): 5 / 27 LOC (18.52%)
- RuboCop (Complaints/Violations):

Offenses:

app/controllers/welcome_controller.rb:2:3: C: Style/EmptyMethod: Put empty method definitions on a single line.
  def index ...
  ^^^^^^^^^
app/models/user.rb:3:3: C: Rails/Validation: Prefer the new style validations validates :column, presence: value over validates_presence_of.
  validates_presence_of :password, require: true
  ^^^^^^^^^^^^^^^^^^^^^
app/models/user.rb:4:22: C: Layout/TrailingWhitespace: Trailing whitespace detected.
  has_secure_password
                     ^
app/models/user.rb:5:1: C: Layout/EmptyLinesAroundClassBody: Extra empty line detected at class body end.

9 files inspected, 4 offenses detected

### CODE REVIEW

File: app/controllers/welcome_controller.rb [hunk 1]
```diff
@@ -1,6 +1,4 @@
 class WelcomeController < ApplicationController
```
Comment (priyapower) [inline app/controllers/welcome_controller.rb:1]: We deleted this?

Comment (priyapower) [reply app/controllers/welcome_controller.rb:1]: Oh never mind - it is white space removal

**Review by priyapower — APPROVED** [review]
Looks awesome!

File: app/controllers/welcome_controller.rb (hunk 1 above)
Comment (Arique1104) [reply app/controllers/welcome_controller.rb:1]: I was a rubocop violation, so I deleted all the white space.

**Pull request merged**